src/main/resources/com/ncc/aif/ontologies/README.md

```diff
@@ -0,0 +1,66 @@
+# AIF Ontologies
+
+The following is a listing of the ontologies found at AIDA-Interchange-Format/src/main/resources/com/ncc/aif/ontologies.
+Each ontology will contain a short description of what they are built from, what they are used for,
+and any other points of relevent information that might be of some use to individuals using these files.
+
+# Differences Between Ontology and OwlOntology
+
+Some ontologies have a corresponding file with a similar name such as SeedlingOntology and SeedlingOwlOntology. In these
+cases, the Owl ontology signifyies that the rdf file is compatible with strict owl, in particular using the correct syntax
+for more complex properties of classes such as RangeIncludes. The non-Owl ontology uses a simpler structure for easier extraction
+of important values.  Both the Owl and non-Owl ontology should define the same vocabulary.
+
+## AidaDomainOntologiesCommon
+
+A collection of top level classes most other ontologies reference as the highest level class.  Additionally, there are
+also a few universal classes that include important properties such as the CanHaveNumericValue class.  This ontology was
+created by individually selecting those class values and not from any other source of data.
+
+## EntityOntology
+
+A collection of Entity classes determined by the Ontology working group.  Entity referring to
+a physical being or intangible concept that either interacts or is subject to interaction.  This in combination with Events
+and Relations ontologies contain the vocabulary that will be used for defining messages post-M18 evaluation.
+
+## EventOntology
+
+A collection of Event classes determined by the Ontology working group.  Event referring to
+an experience that occurs when two or more Events interact in some way.  Some examples being building weaponry, signing a
+treaty, and the moving of people from one place to another. This in combination with Entity
+and Relations ontologies contain the vocabulary that will be used for defining messages post-M18 evaluation.
+
+## InterchangeOntology
+
+The main purpose of this ontology is to contain the Interchange vocabulary and to be used by performers using strict OWL
```

Comment:
I think "The main purpose of this ontology is to contain the Interchange vocabulary" is correct, but I'm not sure that the rest is correct.  Let's discuss to help me understand this statement, and the following sentence starting with "In particular".